CIP-0062/README.md

```diff
@@ -299,6 +265,18 @@ The API chosen here is for the minimum API necessary for dApp <-> Wallet
 interactions without convenience functions that don't strictly need the wallet's
 state to work.
 
+#### api.getVotingPurposes(): Promise\<VotingPurpose>[]
+
+Errors: [`APIError`](#extended-apierror)
+
+This is a chance for dApps to ask wallets what `VotingPurpose`s the user wishes to allow the dApp access to. It is up to wallet implementors if they deem it necessary to prompt the user for their permission to share supported Purposes.
```

Comment:
As this is just defined as an endpoint to fetch supported purposes by the wallet, there is no access request needed. 

Else endpoint would need to be redefined as `enableVotingPurposes(purpose: VotingPurpose[])` that takes a list of purposes to enable just like previous `catalyst.enable(purpose: VotingPurpose[])` call. 

But I dont see this as needed, I happy with wallet just returning a list of purposes it support. Its enough that the user ask for CIP62 permission once, when calling CIP30 enable(). Else it will result in a bad UX.

So text here should be updated to remove info about extra access request.

Reply:
@Scitz0 

I agree -- change made.